src/input/keyboard.c

```diff
@@ -386,11 +386,33 @@ handle_key_release(struct server *server, uint32_t evdev_keycode)
 		end_cycling(server);
 	}
 
+	key_state_bound_key_remove(evdev_keycode);
+
+	/*
+	 * There are some clients (for example blender) that want to see the
+	 * modifier-release-event even if it was part of a keybinds. This is
+	 * treated as a special case and can only be achieved by configuration.
+	 *
+	 * Most clients (including those using Qt and GTK) are setup to not see
+	 * these modifier release events - and actually misbehave if they do.
+	 * For example Firefox shows the menu bar if alt is pressed and then
+	 * released, whereas if only pressed (because the release is absorbed)
+	 * nothing happens. So, if Firefox saw bound modifier-release-events it
+	 * would show the menu bar every time the window-switcher is used with
+	 * alt-tab.
+	 */
+	bool rc_send_absorbed_modifier_release_events_to_client = true;
```

Comment:
I guess this could be a window rule property then?

The only other solution to this that I can see is delaying modifier press signals until either a release or a press of the next non-modifier key (and then send press events for both if its not a keybind or don't send anything in case it is). That has its own issues though.. e.g. consider a game where you press `CTRL` to crouch not  actually crouching until you also press some other key.

Reply:
I think this is one of the reasons the SUPER/windows key exists, to separate it from other software using key modifiers like ALT, SHIFT, CTRL, etc... is that not the case?

Reply:
I say this because I use the SUPER key for keybinds, in that way it doesn't interfere in applications.

Reply:
I see your point but key-combos like `A-Tab`, `A-F4` and similar are kind of a standard for decades. In a perfect world I'd agree that compositors / window managers should only use super + key combinations and also be the only ones allowed to use them. That is not the world we live in though.